app/controllers/casework/decisions_controller.rb

```diff
@@ -2,11 +2,7 @@ module Casework
   class DecisionsController < Casework::BaseController
     include SetDecisionAndAuthorise
 
-    before_action :set_decision, except: [:create, :index]
-
-    def index
```

Comment:
This page has been replaced by the send_decisions_form page which lists the decisions above the form.